src/set_theory/zfc/ordinal.lean

```diff
@@ -16,59 +16,127 @@ Further development can be found on the branch `von_neumann_v2`.
 
 ## Definitions
 
-- `Set.is_transitive` means that every element of a set is a subset.
+- `Set.transitive`: every element of a set is a subset.
+- `Set.is_ordinal`: a set is hereditarily transitive.
 
 ## Todo
 
-- Define von Neumann ordinals.
 - Define the basic arithmetic operations on ordinals from a purely set-theoretic perspective.
 - Prove the equivalences between these definitions and those provided in
   `set_theory/ordinal/arithmetic.lean`.
 -/
 
 universe u
 
-variables {x y z : Set.{u}}
+variables {x y z w : Set.{u}}
+
+local attribute [simp] subtype.coe_inj
 
 namespace Set
 
+/-! ### Transitive sets -/
+
 /-- A transitive set is one where every element is a subset. -/
-def is_transitive (x : Set) : Prop := ∀ y ∈ x, y ⊆ x
```

Comment:
I support the rename, but can that be done separately so the diff is simplified?